### ISSUE TITLE
E-mail addresses should not be auto-corrected

### ISSUE THREAD

**Issue by ARitz-Cracker** [dslul/openboard#425]
**Describe the bug**
E-mail addresses are being auto-corrected as "normal" words

**To Reproduce**
1. Go to any text input e-mail that expects an e-mail address, a website with a login page for example.
2. Attempt to type in "aritz@aritzcracker.ca"
3. Experience mild rage as the string gets corrected to "a ritzy a Ritz cracker" after hitting the "." key
4. Experience more mild rage as ".ca" gets corrected to "can"

**Expected behavior**
Either auto-correct should be disabled when the keyboard is inputting an e-mail address, or words with "@" symbols in them should be treated differently.

**Smartphone (please complete the following information):**
 - Device: OnePlus 5t
 - OS: LineageOS 18.1

**Comment by Pentaphon** [comment]
> I want to add domains to this too.

Yes please! I keep getting this and I don't know why this is the default behavior on AOSP's keyboard code.

**Comment by hamishmb** [comment]
I keep getting this with email addresses too, it's rather annoying.

**Comment by dabat** [comment]
Fixing this will be a great improvement.
Please and thank you.
